browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2303,11 +2303,10 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			controls['button'] = button;
 			if (builder.options.noLabelsForUnoButtons !== true) {
-				var label = L.DomUtil.create('label', 'ui-content unolabel', button);
-				label.htmlFor = buttonId;
-				label.textContent = builder._cleanText(data.text);
-				builder._stressAccessKey(label, button.accessKey);
-				controls['label'] = label;
+				var span = L.DomUtil.create('span', 'ui-content unolabel', button);
```

Comment:
Are we sure we want to remove native "label" type?
It provides automatic information for text-to-speach software what to read and which input filed it labels

Reply:
I see it's not for button element: https://www.w3schools.com/tags/tag_label.asp

Reply:
but don't we need to add 'labelled-by' or 'labelFor' then?

Reply:
It does not specify the attribute change in the task description. Please ask Lilly for more details